### ISSUE TITLE
`python3-zxcvbn` is not maintain in Debian

### ISSUE THREAD

**Issue by ikus060** [dwolfhub/zxcvbn-python#76]
I want to make my application distributed on Debian, but I found out zxcvbn package in not maintain on Debian Trixie.

https://tracker.debian.org/pkg/python-zxcvbn

Could any body take over ?

**Comment by bgermann** [comment]
The python-zxcvbn package was removed from Debian. Please have a look at https://tracker.debian.org/pkg/python-zxcvbn-rs-py

**Comment by bgermann** [comment]
As this is not an issue with this software you might want to close this bug.

**Comment by dwolfhub** [comment]
I don't think I ever maintained it, but if someone wants to take it up they can.

**Comment by ikus060** [comment]
With some help, I manage to get zxcvbn package in Debian sid and it should reach Debian Trixie before it's release.

https://packages.debian.org/sid/python3-zxcvbn

**Comment by bgermann** [comment]
No, it will not enter trixie without an exception from the release team. The freeze is already in place and new packages are not entering anymore.
